unbxdAutosuggest.js

```diff
@@ -1224,10 +1237,29 @@ var unbxdAutoSuggestFunction = function ($, Handlebars, undefined) {
 			if (!this.options.popularProducts.title) {
 				this.options.popularProducts.title = 'autosuggest';
 			}
+			if (!this.options.popularProducts.additionalFields) {
+				this.options.popularProducts.additionalFields = [];
+			}
+		}
+		, getPopularProductFields: function () {
+			this.options.popularProductsFields.push(this.options.popularProducts.title);
+			if (this.options.popularProducts.price && typeof this.options.popularProducts.priceFunctionOrKey == "string"
+				&& this.options.popularProducts.priceFunctionOrKey) {
+				this.options.popularProductsFields.push(this.options.popularProducts.priceFunctionOrKey);
+			}
+			if (this.options.popularProducts.image) {
+				if (typeof this.options.popularProducts.imageUrlOrFunction == "string"
+					&& this.options.popularProducts.imageUrlOrFunction) {
```

Comment:
Isn't this suppose to be in reverse order:`if (this.options.popularProducts.imageUrlOrFunction && typeof this.options.popularProducts.imageUrlOrFunction == "string")`